src/vivarium_public_health/population/base_population.py

```diff
@@ -175,20 +176,41 @@ def get_demographic_proportions_for_creation_time(
         ]
 
 
-class AgeOutSimulants:
+class AgeOutSimulants(VivariumComponent):
     """Component for handling aged-out simulants"""
 
+    def __repr__(self) -> str:
+        return "AgeOutSimulants()"
+
+    ##############
+    # Properties #
+    ##############
+
     @property
     def name(self) -> str:
         return "age_out_simulants"
 
+    @property
+    def columns_required(self) -> List[str]:
+        """A list of the columns this component requires that it did not create."""
+        return self._columns_required
+
+    #####################
+    # Lifecycle methods #
+    #####################
+
+    def __init__(self):
+        super().__init__()
+        self._columns_required = []
+
     # noinspection PyAttributeOutsideInit
     def setup(self, builder: Builder) -> None:
-        if builder.configuration.population.exit_age is None:
-            return
-        self.config = builder.configuration.population
-        self.population_view = builder.population.get_view(["age", "exit_time", "tracked"])
-        builder.event.register_listener("time_step__cleanup", self.on_time_step_cleanup)
+        if builder.configuration.population.exit_age is not None:
+            self.config = builder.configuration.population
+            self._columns_required.extend(["age", "exit_time", "tracked"])
```

Comment:
Is this behaving as intended? This extends the required cols whereas it used to re-define them.

Reply:
oh wait, nm they're always just empty at this point so effectively the same thing

Reply:
It used not to define `self.population_view` at all, since there was no need for one. Now we are setting it to be None, rather than not existing at all. 

What I'm doing here is adding these columns to be required if there is an exit age (which will make it so that a population view is created with those columns when we get to the `super` call below.